typed-builder-macro/src/struct_info.rs

```diff
@@ -85,15 +87,27 @@ impl<'a> StructInfo<'a> {
                 empty_type()
             }
         }));
-        let init_fields_expr = self.included_fields().map(|f| {
-            f.builder_attr.via_mutators.as_ref().map_or_else(
-                || quote!(()),
-                |via_mutators| {
-                    let init = &via_mutators.init;
-                    quote!((#init,))
-                },
-            )
-        });
+        let builder_method_const = Rc::new(OnceCell::new());
+        let init_fields_expr = self
+            .included_fields()
+            .map({
+                let builder_method_const = builder_method_const.clone();
+                move |f| {
+                    f.builder_attr.via_mutators.as_ref().map_or_else(
```

Comment:
Does no one use simple `if let`s nowadays?

Reply:
If you're about the `matches!` macro, then there would be a dangling branch, because it needs an inverted pattern — not `Lit(_)`

Reply:
No, it's about the `map_or_else`. But I just noticed that it was added by someone else, so nevermind.